base_classes/NXbeam.nxdl.xml

```diff
@@ -170,31 +170,48 @@
             <dim index="1" value="m"/>
         </dimensions>
     </field>
-    <field name="incident_polarization" type="NX_NUMBER" units="NX_ANY"
-        deprecated="use incident_polarization_stokes, see https://github.com/nexusformat/definitions/issues/708">
+    <field name="incident_polarization" type="NX_NUMBER" units="NX_ANY">
         <doc>Polarization vector on entering beamline component</doc>
         <dimensions rank="2">
             <dim index="1" value="nP"/>
             <dim index="2" value="2"/>
         </dimensions>
     </field>
-    <field name="final_polarization" type="NX_NUMBER" units="NX_ANY"
-        deprecated="use final_polarization_stokes, see https://github.com/nexusformat/definitions/issues/708">
+    <field name="final_polarization" type="NX_NUMBER" units="NX_ANY">
         <doc>Polarization vector on leaving beamline component</doc>
         <dimensions rank="2">
             <dim index="1" value="nP"/>
             <dim index="2" value="2"/>
         </dimensions>
     </field>
     <field name="incident_polarization_stokes" type="NX_NUMBER" units="NX_ANY">
-        <doc>Polarization vector on entering beamline component using Stokes notation</doc>
+        <doc>
+             Polarization vector on entering beamline component using Stokes notation
+
+             The Stokes parameters are four components labelled I,Q,U,V or S_0,S_1,S_2,S_3.
+             These are defined with the standard Nexus coordinate frame unless it is
+             overridden by an NXtransformations field pointed to by a depends_on attribute.
+             The last component, describing the circular polarization state, is positive
+             for a right-hand circular state - that is the electric field vector rotates
+             clockwise at the sample and over time when observed from the source.
+
+             I is the beam intensity (often normalized to 1), and Q, U, and V are derived
+             from p (the fractional degree of polarization), and the elliptical components
+             Ψ and Χ (psi and chi). Ψ is the orientation angle of the polarization ellipse,
+             relative to the x axis and increasing counter-clockwise around z. Χ is the
+             ellipticity angle, or the arctan of the ratio of the minor/major axes of the
+             polarization ellipse.
```

Comment:
The references to Psi & Chi & 'p' are useful background info, but are not used in NXbeam or elsewhere in NeXus. I recommend to instead state the sign conventions of S1..S3 (Q, U, V).

```suggestion
              I (S_0) is the beam intensity (often normalized to 1). Q, U, and V scale 
              linearly with the total degree of polarization, and indicate the relative
              magnitudes of the pure linear and circular orientation contributions.
              
              Q (S_1) is linearly polarized along the x axis (Q > 0) or y axis (Q < 0).
              
              U (S_2) is linearly polarized along the x==y axis (U > 0) or the
              -x==y axis (U < 0). 

              V (S_3) is circularly polarized. V > 0 when the electric field vector rotates
              clockwise at the sample with respect to time when observed from the source; 
              V < 0 windicates the opposite rotation.
```

Reply:
Looks great, thanks!  Reviewed in code camp with @PeterC-DLS